lib/vanagon/platform/windows.rb

```diff
@@ -170,7 +162,7 @@ def output_dir(target_repo = "")
       #
       # @return [String] the cygwin drive root
       def drive_root
-        "/cygdrive/c"
+        "C:"
```

Comment:
The only reason I would see to keep them is if they can be overridden from a platforms file, and someone wants to setup a build environment on a drive besides `C:`. I suspect that would take more refactoring to make possible.